src/libraries/System.Private.CoreLib/src/System/Buffers/ConfigurableArrayPool.cs

```diff
@@ -72,7 +72,6 @@ public override T[] Rent(int minimumLength)
                 return Array.Empty<T>();
             }
 
-            ArrayPoolEventSource log = ArrayPoolEventSource.Log;
```

Comment:
This has potential to introduce regression. Is this change an improvement or at least a wash?

Reply:
I'll revert it if there's any concern: it's orthogonal to the main purpose of this change.